holmes/plugins/toolsets/http/http_toolset.py

```
@@ -0,0 +1,516 @@
+            raise RuntimeError(
+                "HTTP toolset not configured. Call prerequisites_callable first."
+            )
+        return self._http_config
+
+    def match_endpoint(
+        self, url: str
+    ) -> Tuple[Optional[EndpointConfig], Optional[str]]:
+        """Find the endpoint config that matches the given URL."""
+        try:
+            parsed = urlparse(url)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)